src/review_gator/review_gator.py

```diff
@@ -452,6 +462,16 @@ def get_mps(repo, branch, output_directory=None):
                 if comment is not None:
                     result = comment.vote
                     review_date = comment.date_created
+
+                    src_git_repo = mp.source_git_repository_link.replace(
+                        'https://api.launchpad.net/devel/',
+                        'https://git.launchpad.net/')
+
+                    tmpdir = tempfile.TemporaryDirectory()
```

Comment:
As far as I can see you aren't explicitly cleaning up tmpdir. These tmpdirs would slowly accumulate in the /tmp directory which would be cleaned up on reboot, I don't think we ever intentionally reboot our nodes though.

you could wrap that piece of code in a context manager so it is explicitly cleaned up:
`with tempfile.TemporaryDirectory() as tempdir:`

Reply:
So actually, the tempdir is removed when the python context (i.e. function scope) is gone: https://docs.python.org/3/library/tempfile.html

This was a bit of a gotcha for me, as I originally put the call in the `get_git_repo` function, but it kept getting deleted when that function returned.

Reply:
Ah I didn't know that, that is really interesting